libs/components/packages/src/schematics/utility/json-file.ts

```diff
@@ -48,13 +48,13 @@ export class JsonFile {
       );
     }
 
-    return this._jsonAst;
+    return this.#_jsonAst;
   }
 
-  private _jsonAst: Node | undefined;
+  #_jsonAst: Node | undefined;
 
-  constructor(private readonly host: Tree, private readonly path: string) {
-    const buffer = this.host.read(this.path);
+  constructor(readonly host: Tree, readonly path: string) {
```

Comment:
I think you can remove the `readonly` as well.

Reply:
Fixed